.changeset/tidy-lies-confess.md

```diff
@@ -0,0 +1,14 @@
+---
+"@razorpay/blade": minor
+---
+
+feat(ActionList): add Virtualization in ActionList
+
+```jsx
+<ActionList isVirtualized>
+</ActionList>
+```
+
+> [!NOTE]
+>
+> Current version only supports virtulization of fixed height list where items do not have descriptions. We'll be adding support for dynamic height lists in future versions
```

Comment:
can we add this in docs as well ?

Reply:
Added a note in ActionList docs prop table. Will add better note when we write documentation for virtulization